tests/test_base_models.py

```diff
@@ -1259,10 +1259,23 @@ def run_tests():
     unittest.main()
     
 def main():
+    import cProfile, pstats
```

Comment:
Did we decide to leave this for another PR?

Reply:
Let's move it to another PR